src/RouteJs/RouteJs.cs

```diff
@@ -35,6 +42,21 @@ IEnumerable<IConstraintsProcessor> constraintsProcessors
 		}
 
 		/// <summary>
+		/// Gets or sets whether urls are converted to lower case.
+		/// </summary>
+		public static bool LowerCaseUrls { get; set; }
```

Comment:
Perhaps we should have a .NET 4.5 build in addition to the .NET 4.0 build, and then gate the code with a `#ifdef`